build.gradle

```diff
@@ -162,6 +153,12 @@ subprojects {
             }
         }
 
+        configurations.all {
+            resolutionStrategy.dependencySubstitution {
+                substitute module('org.jsoup:jsoup:1.14.3') using module('org.jsoup:jsoup:1.15.3')
```

Comment:
Do you need to specify the original version here?
```
substitute module('org.jsoup:jsoup') using module("org.jsoup:jsoup:$jsoupVersion")
```

Reply:
your right re version , updated,

this is brought in via dokka kdoc generation dependency, so I don't think this will be ending up in any artifact.

we can not specifically only scan the runtimeClasspath of corda-api as we do in other projects, as snyk falls over due to this configuration not being available in all submodules as logged here by another user https://github.com/snyk/snyk-gradle-plugin/issues/192